liberty-remote/pom.xml

```diff
@@ -150,6 +185,11 @@
       <groupId>org.jboss.arquillian.container</groupId>
       <artifactId>arquillian-container-test-spi</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.arquillian.protocol</groupId>
+      <artifactId>arquillian-protocol-rest-jakarta</artifactId>
+      <version>1.7.0.Alpha11</version>
```

Comment:
The other dependencies here do not have a version (I believe they get it from the parent pom?). Should this one have the version here?

Reply:
This is because of issue https://github.com/arquillian/arquillian-core/issues/418.  When that is merged in and Alpha12 is created the version will go away.  The plan is to do that by end of day so I will update the PR tonight or tomorrow morning when Alpha12 is created and then this version will be removed.  At that point I believe this will be ready to merge in.